src/web/mjs/connectors/MangaNeloInfo.mjs

```diff
@@ -1,31 +1,85 @@
-import MangaNel from './MangaNel.mjs';
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
 
-export default class MangaNeloInfo extends MangaNel {
+export default class MangaNeloInfo extends Connector {
 
     constructor() {
         super();
         super.id = 'manganeloinfo';
         super.label = 'MangaNeloInfo';
         this.tags = [ 'manga', 'webtoon', 'english' ];
-        this.url = 'http://manganelo.info';
+        this.url = 'https://manganelo.info';
 
-        this.path = '/?page=';
-        this.queryMangaTitle = 'div.manga-info-top ul.manga-info-text li h1';
-        this.queryMangasPageCount = 'div.pagination ul.pagination li:nth-last-of-type(2) a';
-        this.queryMangas = 'div.doreamon div.first ul li h3 a';
-        // NOTE: a corresponding entry for chapter/page queries must be defined in the base class (required for cross-domain-support)
+        this.path = '/genre';
+        this.mangaTitleFilter = /(\s+manga|\s+webtoon|\s+others)+\s*$/gi;
+        this.chapterTitleFilter = /^\s*(\s+manga|\s+webtoon|\s+others)+/gi;
+        this.queryMangaTitle = 'section div.flex div.grow h1';
+        this.queryMangas = 'main div.gap-4 div.gap-y-3 div.space-y-3 div.grid div.gap-2 div.overflow-hidden h3 a';
```

Comment:
Hum, too long and ugly css prone to break (too long with generic class names)

`div.grid h3.truncate a`